pipeline.Makefile

```diff
@@ -109,7 +117,34 @@ schema-help:
 	@:$(info =================)
 	@:$(info $(DEBUG))
 
+.PHONY: validate-schema
+validate-schema: $(SCHEMA_FILE)
+	@mkdir -p $(VALIDATOR_OUTPUT_DIR)
+	@echo "Validating schema $(SCHEMA_FILE)..."
+	@$(RUN) linkml validate --schema $< > $(SCHEMA_VALIDATE_LOG) 2>&1 && \
+		echo "Schema validation passed." >> $(SCHEMA_VALIDATE_LOG) || \
+		echo "Schema validation failed." >> $(SCHEMA_VALIDATE_LOG)
+	@echo "  Log written to $(SCHEMA_VALIDATE_LOG)"
+
 
-# .PHONY: validate-schema
-# validate: $(SCHEMA_FILE)
-# 	$(RUN) linkml-validate -s $< -C $(INPUT_FILES)
+.PHONY: validate-data
+validate-data: $(SCHEMA_FILE)
+	@:$(call check_input_files)
+	@mkdir -p $(VALIDATOR_OUTPUT_DIR)
+	@rm -f $(DATA_VALIDATE_LOG)
+	@for f in $(INPUT_FILES); do \
+		class=$$(basename $$f | sed -E 's/\.[ct]sv$$//' | tr '-' '_' | tr '[:upper:]' '[:lower:]'); \
+		out="$(VALIDATOR_OUTPUT_DIR)/$$(basename $$f).validate.log"; \
+		echo "Validating $$f as class '$$class'..." | tee -a $(DATA_VALIDATE_LOG); \
+		if $(RUN) linkml validate --schema $(SCHEMA_FILE) --target-class $$class $$f 2>>$(DATA_VALIDATE_LOG); then \
+			echo "  ✓ $$f passed." | tee -a $(DATA_VALIDATE_LOG); \
+		else \
+			echo "  ✗ $$f failed. See $$out" | tee -a $(DATA_VALIDATE_LOG); \
+		fi; \
+	done
+	@echo "Validation summary written to $(DATA_VALIDATE_LOG)"
+
+.PHONY: validate-debug
+validate-debug:
```

Comment:
Is this necessary, since it's the same as `schema-debug`?

Reply:
No it's not necessary. It's there as a stub in case we start  to have different validate variables. I'm actually thinking about changing all of this anyway... having a series of schema- and data-. What do you think?